sequins.go

```diff
@@ -132,14 +132,26 @@ func (s *sequins) initCluster() error {
 	if err != nil {
 		return err
 	}
+	
+	ip := ""
+	ipAddresses, err := net.LookupHost(hostname)
+	if err	!= nil {
```

Comment:
My editor should go fmt, but I guess it didn't pushing change.  We don't currently validate gofmt.